tests/commands/device/supported.spec.ts

```diff
@@ -30,7 +20,7 @@ describe('balena devices supported', function() {
 
 		const { out, err } = await runCommand('devices supported -h');
 
-		expect(cleanOutput(out)).to.deep.equal(cleanOutput([HELP_RESPONSE]));
+		expect(cleanOutput(out)).to.contain('$ balena devices supported');
```

Comment:
Yeah, I think the help text testing is maybe a bit over the top at the moment.  This is a more reasonable basic check that it's working.

Reply:
Yes, I kept having to amend the test for any small changes to the description, so I thought of making this change. :-)